sdk/feature-set/src/lib.rs

```diff
@@ -12,853 +12,855 @@
 //!      through these steps, the PR process will facilitate a keypair holder being picked. That
 //!      person will generate the keypair, provide pubkey for PR, and ultimately enable the feature.
 //! 2. Add a public module for the feature, specifying keypair pubkey as the id with
-//!    `solana_sdk::declare_id!()` within the module.
+//!    `solana_program::declare_id!()` within the module.
 //!    Additionally, add an entry to `FEATURE_NAMES` map.
 //! 3. Add desired logic to check for and switch on feature availability.
 //!
 //! For more information on how features are picked up, see comments for `Feature`.
+#![cfg_attr(RUSTC_WITH_SPECIALIZATION, feature(min_specialization))]
 
 use {
     lazy_static::lazy_static,
-    solana_program::{epoch_schedule::EpochSchedule, stake_history::Epoch},
-    solana_sdk::{
+    solana_program::{
         clock::Slot,
+        epoch_schedule::EpochSchedule,
```

Comment:
Once EpochSchedule gets moved out of `solana_program`, this crate could depend only on pubkey, hash, clock, and that :partying_face: